async-http-client-backend/zio/src/test/scala/sttp/client3/asynchttpclient/zio/AsyncHttpClientZioWebSocketTest.scala

```diff
@@ -8,21 +8,25 @@ import sttp.client3.impl.zio.{RIOMonadAsyncError, ZioTestBase, ZioWebSockets}
 import sttp.client3.testing.ConvertToFuture
 import sttp.monad.MonadError
 import sttp.ws.WebSocketFrame
+import zio.Clock.ClockLive
 import zio.stream._
-import zio.{Clock, Schedule, Task, ZIO, durationLong}
+import zio.{Schedule, Task, ZIO, durationLong}
 
 import scala.concurrent.duration.FiniteDuration
 
 class AsyncHttpClientZioWebSocketTest extends AsyncHttpClientWebSocketTest[Task, ZioStreams] with ZioTestBase {
   override val streams: ZioStreams = ZioStreams
 
   override val backend: SttpBackend[Task, WebSockets with ZioStreams] =
-    runtime.unsafeRun(AsyncHttpClientZioBackend())
+    unsafeRun(AsyncHttpClientZioBackend())
   override implicit val convertToFuture: ConvertToFuture[Task] = convertZioTaskToFuture
   override implicit val monad: MonadError[Task] = new RIOMonadAsyncError
 
   override def eventually[T](interval: FiniteDuration, attempts: Int)(f: => Task[T]): Task[T] = {
-    ZIO.sleep(interval.toMillis.millis).flatMap(_ => f).retry(Schedule.recurs(attempts)).provideLayer(Clock.live)
+    ClockLive
```

Comment:
I think `Clock.sleep` is actually better because it gives to the user the option to use `TestClock` instead in their tests.